libs/pages/application/src/lib/ui/page-settings-general/page-settings-general.tsx

```diff
@@ -77,21 +83,20 @@ export function PageSettingsGeneral(props: PageSettingsGeneralProps) {
             <JobGeneralSettings
               isEdition={true}
               jobType={isCronJob(type) ? ServiceTypeEnum.CRON_JOB : ServiceTypeEnum.LIFECYCLE_JOB}
-              organization={props.organization}
+              organization={organization}
             />
           )}
           {isApplication(type) && (
             <>
               <EditGitRepositorySettingsFeature />
-              <BlockContent title="Build mode">
+              <BlockContent classNameContent="gap-3 flex flex-col" title="Build & deploy">
```

Comment:
should it stay "build mode" no?

Reply:
it should be discuss with @alecarrano when he'll come back from vacation